pkg/tinyusb/hw/hw_sam0.c

```diff
@@ -48,7 +48,9 @@ int tinyusb_hw_init(void)
     GCLK->CLKCTRL.reg = GCLK_CLKCTRL_CLKEN
                         | GCLK_CLKCTRL_GEN(sam_usbdev_config[0].gclk_src)
                         | GCLK_CLKCTRL_ID(USB_GCLK_ID);
+#if MODULE_PM_LAYERED
     pm_block(SAMD21_PM_IDLE_1);
+#endif /* MODULE_PM_LAYERED */
```

Comment:
Is this needed? We already have

```C
#ifdef MODULE_PM_LAYERED
void pm_block(unsigned mode);
#else
static inline void pm_block(unsigned mode) { (void)mode; }
#endif
```

Reply:
I dont't know, probably there was compilation error and I had to add it for the `riotboot` compilation, problably because `periph_pm` is not used in bootloader. Let me check again.

Reply:
Indeed, including `pm_layered.h` instead of `periph/pm.h` solves the compilation problem. This kind of conditional using `pm_block` and `pm_unblock` is not only used in this file, but in all `pkg/tinyusb/hw/hw_*.c` files as well as in `usbdev_synopsys_dwc2.c` where it initially used in that way. That is, this change would require a more complex clean-up. Should it be done with this PR or as separate PR?

Reply:
I don't follow - why does *not* adding the `#ifdef`s with this PR require more cleanup?

If they can be removed in other places is unrelated.

Reply:
Fixed